app/controller_services/shopping_list_items_controller/create_service.rb

```diff
@@ -24,7 +24,6 @@ def perform
 
       ActiveRecord::Base.transaction do
         item.save!
-        shopping_list.touch
```

Comment:
Should no longer be needed but we'll double check at the point where we make these tests pass again.